src/utils/addExtremeValuesToArr.js

```diff
@@ -0,0 +1,13 @@
+export function addExtremeValuesToArr(values) {
```

Comment:
Mmm, I guess that this just works with intervals, so let's try to find a better name...

Reply:
What do you think?

`fillIntervalValue`
`addIntervalValue`

Reply:
makeClosedInterval ?